### PR TITLE
Frasia

### PR DESCRIPTION
**Pull request by SFrasia**
## Description

Worked on the Login changes.
Created Dashboard with MUI components and Chart.js to showcase the graphs element.

Screen Shot 2024-07-24 at 9 08 49 AM

### CODE REVIEW

File: client/src/Components/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+//         </li>
+//       ))}
+//     </ul>
+//   </aside>
+// )
+// }
+
+// bg-gray-100 text-gray-900 h-screen px-4 fixed w-16 md:w-64 border-r border-gray-300
+// w-56 bg-zinc-900 h-screen border-r border-gray-100
+
```
Comment (otsembo) [inline client/src/Components/Sidebar.jsx:49]: If you are no longer using this you can remove them.

File: client/src/Components/SidebarData.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        title: 'Users',
+        icon: <GroupsIcon />,
+        link:'/users',
+        allowedRoles: ['admin']
+    },
+    // {
+    //     title: 'Permission',
+    //     icon: <ManageAccountsIcon />,
+    //     link:'/permission'
+    // },
```
Comment (otsembo) [inline client/src/Components/SidebarData.jsx:46]: If not using, remove this

**Review by otsembo — APPROVED** [review]
Fantastic job @SFrasia 
Small comments to check but great job.

File: client/src/Components/Sidebar.jsx (hunk 1 above)
Comment (SFrasia) [reply client/src/Components/Sidebar.jsx:49]: Noted

File: client/src/Components/SidebarData.jsx (hunk 2 above)
Comment (SFrasia) [reply client/src/Components/SidebarData.jsx:46]: Noted as well

**Review by b-tess — APPROVED** [review]

**Pull request merged**